includes/admin/views/html-notice-update.php

```diff
@@ -18,16 +18,22 @@
 ?>
 <div id="message" class="updated woocommerce-message wc-connect">
 	<p>
-		<strong><?php esc_html_e( 'WooCommerce data update', 'woocommerce' ); ?></strong> &#8211; <?php esc_html_e( 'We need to update your store database to the latest version.', 'woocommerce' ); ?>
+		<strong><?php esc_html_e( 'Database update required', 'woocommerce' ); ?></strong>
+	</p>
+	<p>
+		<?php
+			esc_html_e( 'WooCommerce has been updated! To keep things running smoothly, we have to update your database to the newest version.', 'woocommerce' );
+
+			/* translators: 1: Link to docs 2: Close link. */
+			printf( ' ' . esc_html__( 'The database update process may take a little while, so please be patient. Advanced users can alternatively update via %1$sWP CLI%2$s.', 'woocommerce' ), '<a href="https://github.com/woocommerce/woocommerce/wiki/Upgrading-the-database-using-WP-CLI">', '</a>' );
```

Comment:
Like the addition of the WP-CLI mention. Would it also be beneficial to have it say it will run the background?